product-web-app/controller/JobSpecificationController.ts

```diff
@@ -0,0 +1,20 @@
+import { Application, Request, Response } from 'express';
+import JobSpecificationService from '../service/JobSpecificationService.js';
+import JobSpecification from '../model/JobSpecification.js';
+/* eslint-disable */
```

Comment:
what are for those eslint disable?

Reply:
I ve problem with lint error because when i implement this suggestion app stopped working